entrypoint.sh

```diff
@@ -12,20 +12,23 @@ if [ -n "$KEY_BUILD" ]; then
 	SIGNED_PACKAGES="y"
 fi
 
-echo "src-link $FEEDNAME $GITHUB_WORKSPACE/" > feeds.conf
-
 if [ -z "$NO_DEFAULT_FEEDS" ]; then
 	sed \
 		-e 's,https://git.openwrt.org/feed/,https://github.com/openwrt/,' \
 		-e 's,https://git.openwrt.org/openwrt/,https://github.com/openwrt/,' \
 		-e 's,https://git.openwrt.org/project/,https://github.com/openwrt/,' \
-		feeds.conf.default >> feeds.conf
+		feeds.conf.default > feeds.conf
 fi
 
+echo "src-link $FEEDNAME $GITHUB_WORKSPACE/" >> feeds.conf
```

Comment:
Could this configuration also be set configurable, for example `${FEEDPATH:-$GITHUB_WORKSPACE/}`? I'm building a package in a single Github repo (https://github.com/ttimasdf/luci-app-xray) with Makefile in package root, and this line breaks it.

Reply:
Maybe, but it's out of the scope of this PR. You may start another one to address it.